ta4j-core/src/main/java/org/ta4j/core/indicators/ChopIndicator.java

```diff
@@ -45,7 +45,7 @@
 	 */
 	public ChopIndicator( TimeSeries timeseries, int ciTimeFrame, int scaleTo ) {
 		super( timeseries );
-        this.atrIndicator = new ATRIndicator( timeseries, ciTimeFrame );
+        this.atrIndicator = new ATRIndicator( timeseries, 1 );	// ATR(1) = Average True Range (Period of 1)
         hvi = new HighestValueIndicator( new MaxPriceIndicator(timeseries), ciTimeFrame );
         lvi = new LowestValueIndicator( new MinPriceIndicator(timeseries), ciTimeFrame );
         this.timeFrame = ciTimeFrame;

```

Comment:
Is ATR(1) *always* used?  If so, this is okay.  If not, it should be added as a parameter.  Also, the test class should include cases that *must* be modified to pass with this change.  If it does not, they should be added.

Reply:
ATR(1) is always used.  It is part of the definition of CI.  So a constant is fine.